teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -6,10 +6,13 @@
    "source": [
```

Comment:
<p>Add docstring description to <code>smiles</code> argument.</p><p> </p><p><code>&nbsp;&nbsp;smiles : str</code></p><p><code>	Canonical SMILES string.</code></p><p><code>&nbsp;&nbsp;threshold : int</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;The threshold of similarity, default 75%. In PubChem, the default threshold is 90%.</code></p><p><code>&nbsp;&nbsp;n_records : int</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;The number of feedback records.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>done</p>